src/Attribute/Normalize.php

```diff
@@ -8,6 +8,9 @@
 use Patchlevel\EventSourcing\Serializer\Normalizer\ArrayNormalizer;
 use Patchlevel\EventSourcing\Serializer\Normalizer\Normalizer;
 
+/**
+ * @deprecated use the specific normalizer as attribute
```

Comment:
I would also maybe add a sentence about custom normalizers that they should add now the attribute to enable them as attributes